tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+from freezegun import freeze_time
+import pytest
+
+
+@pytest.fixture(params=[
+    "2018-03-25T00:00:00",
+    "2018-03-25T01:00:00",
+    "2018-03-25T02:00:00",
+    "2018-03-25T02:30:00",
+    "2018-03-25T03:00:00",
+    "2018-03-25T04:00:00",
```

Comment:
First, take given dates&times + offsets as kick off examples - feel free to narrow or extend them whenever needed. My intention was to help with cheap option (because it is run by a computer and not through programmer's mind) to test current time related stuff.

On `test_struct` example it is seen, that some times/time offsets are making problems while others are not. If we want to speed up the test, we could think of adding "minimal" marker which would be added to subset of parameters and used on default test run. To do so, let me know, which times and time offsets you consider as "minimal" and I shall provide example of marking them properly (I never did it but it shell be rather simple).

Regarding 25. March - do you want to modify the tests each year? Personally I am happy with testing maya for handling of critical time periods in whatever year and would not attempt to make the tests current year dependent.

Anyway, I realize, my data-set is currently missing the other change (in autumn).